ibis/tests/expr/test_decimal.py

```diff
@@ -68,7 +68,7 @@ def test_promote_decimal_type_max():
     [
         (None, None, (None, None)),
         (38, 2, (38, 2)),
-        (16, 2, (38, 2)),
+        (16, 2, (16, 2)),
```

Comment:
We no longer promote decimals to at least precision 38, though we can hardcode certain minimal precision per operation.

Reply:
Restored the original behaviour for `Sum`